zmon_worker_monitor/zmon_worker/notifications/opsgenie.py

```diff
@@ -61,6 +70,7 @@ def notify(cls, alert, teams=None, per_entity=False, include_alert=True, priorit
 
         responsible_team = alert['alert_def'].get('responsible_team', teams[0]['name'])
         msg = message if message else cls._get_subject(alert, include_event=False)
+        desc = description if description else alert['alert_def'].get('description', None)
```

Comment:
I would modify the default value to be ``''`` instead of ``None``.

Also from reviewing [zmon-scheduler code](https://github.com/zalando-zmon/zmon-scheduler/blob/4cb553e1096dfd980239729c1965c4e7b2ec276b/src/main/java/de/zalando/zmon/scheduler/ng/CeleryBody.java#L31-L44), I can see that for now ``alert_def`` will always be missing the ``description``.

Reply:
> Also from reviewing zmon-scheduler code, I can see that for now alert_def will always be missing the description.

Ah. That puts a spanner in the works! 

Would the team agree to including the description into this message?  If not, I will change the description to default to ''.

While this would help us to define everything in one place (just the zmon description),  it may not be a general use-case for other users. I understand if you think that it is preferable to not include a potentially large amount of data in the message.